tests/unit_test/conftest.py

```diff
@@ -0,0 +1,9 @@
+import logging
+
+import pytest
+
+
+@pytest.fixture(autouse=True)
+def configure_logging():
+    logging.root.handlers = []
```

Comment:
I set up the logging that way before because I didn't know much better I guess. Nowadays I would probably just set the logging level (and call `basicConfig` which wouldn't do anything if there are already handlers): https://github.com/elifesciences/sciencebeam-judge/blob/4afba0a691da2f04f26777b1deb44386546dabf3/tests/conftest.py#L6-L10